101-cdn-with-custom-origin/azuredeploy.parameters.json

```diff
@@ -9,7 +9,7 @@
             "value": "GEN-UNIQUE"
         },
         "originUrl": {
-            "value": "www.microsoft.com"
+            "value": "cdndemo.azurewebsites.net"
```

Comment:
revert this change (or use a web site that can't be removed)

Reply:
This was created specifically for CDN for demos and is being used by our current QS.  I went ahead and changed it back.